src/core/jsdoc-tsd-parser.ts

```diff
@@ -350,8 +378,87 @@ export class JSDocTsdParser {
 								}
 								break;
 
+							case "interface":
+								let objectTypeMember = parsedItem as dom.ObjectTypeMember;
+
+								switch ((objectTypeMember as any).kind) {
+									case "function":
+										let functionDeclaration: any = objectTypeMember as any;
+										objectTypeMember = {

```

Comment:
Maybe we should create the method declaration with the dts-dom-module like dom.create.method(...). It's more safe if the API changes in future..